Makefile

```diff
@@ -37,5 +37,12 @@ cover:
 	@echo "+ $@"
 	@go list -f '{{if len .TestGoFiles}}"go test -coverprofile={{.Dir}}/.coverprofile {{.ImportPath}}"{{end}}' $(shell go list ${PKG}/... | grep -v vendor) | xargs -L 1 sh -c
 
+coveralls:
+	@echo "+ $@"
+# Make sure goveralls is installed.
```

Comment:
Format the comments to align with the commands.

Reply:
When I indented the comment, then it was "executed" during 'make coveralls'. e.g. I saw "# Make sure goveralls is installed." in the terminal output...